pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -0,0 +1,45 @@
+[id='con-pantheon-yaml-file_{context}']
+
+= The Pantheon configuration file
+
+In order to use a Git repository to import content to Pantheon, you need to create a configuration file in the root directory of your repository named `pantheon2.yml`. This yaml file indicates to Pantheon that it can interact with files in the repository.
+
+The configuration file enables the following:
+
+* Importing content from a Git repository into Pantheon
+* Uploading modules into Pantheon using the uploader tool
+
+The configuration file defines the following:
+
+`server`:: The URL of the Pantheon server.
+`repository`:: A unique ID to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
+`modules`:: Patterns for modules that will show up in the search results on Pantheon.
+`resources`:: Image files, legal information or other content that is not.
+
+You need to define the values for `server` and `repository`.
+
+.Example configuration file
+
+// [options="nowrap" subs="normal"]
+----
+# Config file for Pantheon
+## server: Pantheon server URL
+## repository: a unique name, which is visible in the user facing URL
+
+## Note: Due to yaml syntax, any filepaths that start with a wildcard must be surrounded in quotes like so:
+# modules:
+#  - '*.adoc'
+
+server: http://pantheon2.int.us-west.aws.prod.paas.redhat.com[]
```

Comment:
The current URL for Pantheon is internal only, so we should probably change this and all other references to a replaceable such as '<server_url>' for the time being.

Reply:
@adahms I think this is a good candidate for an attribute. We can define it as :ProductUrl: (or some such) in the attributes file.

Reply:
Done.

Reply:
@stoobie - we won't have a separate downstream and upstream version right away, but I agree that using an attribute as suggested could be a good approach for now.

Would you be able to add the attribute in the attributes file for this, and make the value a replaceable for now?

Reply:
Done:

:ProductUrl: http://pantheon2.int.us-west.aws.prod.paas.redhat.com[]